base/complex.jl

```diff
@@ -813,7 +833,7 @@ function asin(z::Complex)
     Complex(ξ,η)
 end
 
-function acos(z::Complex{<:AbstractFloat})
```

Comment:
Am I missing something or now this method has the same signature as the method below causing a StackOverflowError?

Reply:
I'm sure I deleted that other method at some point, but will do so here, thanks.